_posts/2020-01-15-weak-passwords.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Weak Passwords
+date: 2020-01-08 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+When I started using 1Password I manually imported the logins I remembered and changed the passwords foe the most important ones. The others were left with relatively weak passwords, but if they got compromised nothing bad would happen. A few days ago I became curious in finding those account and taking a look at them after almost a decade.
+
+## How To Show Weak passwords in 1Password
+
+Open 1Password and click the “Traditional” layout button to see all your data in columns. The button is just left of the search field at the top of the window or you can select it from the View > Layout menu.
+Make sure View > Columns > Password Strength is enabled.
+Now click the Password Strength column header to sort all your Logins by the strength of their passwords.
+
+## Results
+
+The astounding results was finding in the list, apart from the expected old passwords, also passwords that I was forced to create as weak because of restrictions of that specific website password policy. More than that a few of these weak passwords were from banks. Those bank accounts are also protected with two factor authentication so that would not be such of a big concern, but it is telling at how banks are still behind in this area.
+
+

```

Comment:
Bit short....

Reply:
Zzz? I have been coming here for like a week and there's nothing to do...

Reply:
If you have any idea on something to add let me know =) But yeah I generally tend to write short stuff